posthog/migrations/0309_add_starter_dashboard_template.py

```diff
@@ -0,0 +1,156 @@
+            },
+            {
+                "name": "Referring domain (last 14 days)",
+                "type": "INSIGHT",
+                "color": "black",
+                "filters": {
+                    "events": [{"id": "$pageview", "math": "dau", "type": "events"}],
+                    "display": "ActionsBarValue",
+                    "insight": "TRENDS",
+                    "interval": "day",
+                    "breakdown": "$referring_domain",
+                    "date_from": "-14d",
+                    "breakdown_type": "event",
+                },
+                "layouts": {
+                    "sm": {"h": 5, "w": 6, "x": 0, "y": 10, "minH": 5, "minW": 3},
+                    "xs": {"h": 5, "w": 1, "x": 0, "y": 20, "minH": 5, "minW": 3},
+                },
+                "description": "Shows the most common referring domains for your users over the past 14 days.",
+            },
+            {
+                "name": "Pageview funnel, by browser",
+                "type": "INSIGHT",
+                "color": "green",
+                "filters": {
+                    "events": [
+                        {
+                            "id": "$pageview",
+                            "type": "events",
+                            "order": 0,
+                            "custom_name": "First page view",
+                        },
+                        {
+                            "id": "$pageview",
+                            "type": "events",
+                            "order": 1,
+                            "custom_name": "Second page view",
+                        },
+                        {
+                            "id": "$pageview",
+                            "type": "events",
+                            "order": 2,
+                            "custom_name": "Third page view",
+                        },
+                    ],
+                    "layout": "horizontal",
+                    "display": "FunnelViz",
+                    "insight": "FUNNELS",
+                    "interval": "day",
+                    "breakdowns": [{"type": "event", "property": "$browser"}],
+                    "exclusions": [],
+                    "breakdown_type": "event",
+                    "funnel_viz_type": "steps",
+                },
+                "layouts": {
+                    "sm": {"h": 5, "w": 6, "x": 6, "y": 10, "minH": 5, "minW": 3},
+                    "xs": {"h": 5, "w": 1, "x": 0, "y": 25, "minH": 5, "minW": 3},
+                },
+                "description": "This example funnel shows how many of your users have completed 3 page views, broken down by browser.",
+            },
+        ],
+        tags=[],
+    )
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0308_add_indirect_person_override_constraints"),
+    ]
+
+    operations = [migrations.RunPython(create_starter_template)]
```

Comment:
```suggestion
    operations = [migrations.RunPython(create_starter_template, reverse_code=migrations.RunPython.noop)]
```

RunPython throws when reversed if `reverse_code` is not provided. We _should_ be fine to have a no-op reversal instead